app/src/main/java/org/p2p/wallet/auth/analytics/CreateWalletAnalytics.kt

```diff
@@ -0,0 +1,49 @@
+package org.p2p.wallet.auth.analytics
+
+import org.p2p.wallet.common.analytics.Analytics
+
+private const val ONBOARD_START_BUTTON = "Onboarding_Start_Button"
+private const val ONBOARD_CREATE_PHONE_SCREEN = "Creation_Phone_Screen"
+private const val ONBOARD_CREATE_PHONE_CLICK_BUTTON = "Create_Phone_Click_Button"
+private const val ONBOARD_CREATE_SMS_SCREEN = "Create_Sms_Screen"
+private const val ONBOARD_CREATE_SMS_VALIDATION = "Create_Sms_Validation"
+private const val ONBOARD_CREATE_WALLET_CONFIRM_PIN = "Create_Confirm_Pin"
```

Comment:
Mb `ONBOARDING`?

Reply:
yeah, ok
it was just the very old previous naming that i didn't change